spec/factories/users.rb

```diff
@@ -4,5 +4,8 @@
   factory :user do
     email { Faker::Internet.email }
     password { Faker::Internet.password(min_length: 6) }
+    name { Faker::Name.first_name }
+    birth { Faker::Date.birthday(min_age: 18) }
+    username { Faker::Internet.username }
```

Comment:
for this we have to make sure that the usernames generated by the factory, are unique. In order to do that you can 2 it in 2 different ways:
1. Use [faker unique modifier](https://github.com/faker-ruby/faker#ensuring-unique-values)
  - **Pros**: it's easy to use
  - **Cons**: it's slow, and it can run out of unique values at some point.
2. [Use a sequence](https://github.com/thoughtbot/factory_bot/blob/main/GETTING_STARTED.md#global-sequences):
  -  **Pros**: Fast, and reliable
  - **Cons**: It's more difficult to use